### PR TITLE
Updated Documentation of REMREM Publish Split

### PR DESCRIPTION
**Pull request by Jibandeep**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,9 +70,16 @@ <h2 id="project_tagline"></h2>
 <hr>
 
 <h2>
+<a id="usage-common" class="anchor" href="#usage-common" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Usage Common</h2>
+
+  <p>REMREM Common is a common repository of codes which helps both CLI and service to validate eiffel messages and publish them on RabbitMQ. It cannot be run as micro services or stand alone versions. It just help in proper working of both CLI and service. </p>
```
Comment (SantoshNC68) [inline index.html]: I think it should be made clear, we are talking about publish? REMREM Common can be changed to publish common?

File: index.html [hunk 2]
```diff
@@ -39,6 +39,18 @@ <h2 id="project_tagline"></h2>
 <hr>
 
 <h2>
+<a id="components" class="anchor" href="#components" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Components</h2>
+
+<ul>
+<li>Common</li>
+<li>CLI (Command Line Interface)</li>
+<li>Service</li>
```
Comment (evasiba-ericsson) [inline index.html]: The same here.
Prepend REMREM Publish to each item in the list.

File: index.html (hunk 1 above)
Comment (evasiba-ericsson) [inline index.html]: I agree change to REMREM Publish Common

File: index.html [hunk 3]
```diff
@@ -51,16 +63,69 @@ <h2 id="project_tagline"></h2>
 <h2>
 <a id="how-to-install" class="anchor" href="#how-to-install" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>How to Install?</h2>
 
-<p>Binary is relased via <a href="https://jitpack.io/#Ericsson/eiffel-remrem-publish/">jitPack</a></p>
+<p>Binary is relased via <a href="https://jitpack.io/#Ericsson/eiffel-remrem-publish">jitPack common</a></p>
```
Comment (SantoshNC68) [inline index.html]: I don't think there is need to mention **common** in the Jitpack

File: index.html [hunk 4]
```diff
@@ -51,16 +63,69 @@ <h2 id="project_tagline"></h2>
...
+			}
+			else{
+				document.getElementById("downloadLink").innerHTML = result;
+				var val = document.getElementsByTagName("text")[2].innerHTML;
+				val = val.trim();
+				urlLink = "https://jitpack.io/com/github/Ericsson/eiffel-remrem-publish/publish-"+type +"\/"+val+"/publish-"+type +"-"+val+"."+extn;
+				var link = document.createElement("a");
+				link.setAttribute('id',"MYLINK");
+				link.href = urlLink;
+				document.body.appendChild(link);
```
Comment (SantoshNC68) [inline index.html]: what is the purpose of adding this to the body, as this code is solely handling the downloading part?

File: index.html [hunk 5]
```diff
@@ -51,16 +63,69 @@ <h2 id="project_tagline"></h2>
...
+}
+</script>
+
+<h3>
+<a id="installation" class="anchor" href="#installation" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Installation (Micro service)</h3>
+
+<p>Start the self executing Publish-cli JAR</p>
+<pre>java -jar publish-cli.jar</pre>
+
+<p>Deploy the self executing Publish-Service WAR in Tomcat to use REST services.</p>
```
Comment (SantoshNC68) [inline index.html]: I guess this should be moved to the bottom of the page, where we are describing the Web application.

**Pull request merged**